packages/block-library/src/post-comments-form/view.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import './runtime/init.js';
+import { store, navigate } from './runtime';
+
+store( {
+	state: {
+		core: {
+			commentsFormError: '',
+		},
+	},
+	actions: {
+		core: {
+			commentsFormSubmission: async ( { event, state } ) => {
+				event.preventDefault();
+
+				state.core.commentsFormError = '';
+
+				const formData = new FormData( event.target );
+
+				const res = await fetch(
+					'http://localhost:8888/wp-comments-post.php',
```

Comment:
This will allow the action to not be hard-coded:

```suggestion
					event.target.action,
```